mappings/net/minecraft/world/gen/noise/NoiseRouter.mapping

```diff
@@ -0,0 +1,58 @@
+CLASS net/minecraft/unmapped/C_fpiyqmft net/minecraft/world/gen/noise/NoiseRouter
```

Comment:
This name doesn't convince me at all

Reply:
hmm yeah, do you have any other proposals?

Reply:
I think we can leave it like this, I don't have any ideas and this name is probably used in some string for its registry